src/pager.c

```diff
@@ -155,6 +155,9 @@ pager_request(struct view *view, enum request request, struct line *line)
 	enum open_flags flags = view_is_displayed(view) ? OPEN_SPLIT : OPEN_DEFAULT;
 	int split = 0;
 
+	if (request == REQ_EDIT)
+		return diff_common_edit(view, request, line);
```

Comment:
Seems like a good change, thanks. This has bothered me occasionally in the past, although I hardly use the pager view (perhaps because of that?).

One thing that's weird in `git log -p | tig` is that if my cursor is inside a commit message,
`:edit` will work - searches upwards for the last file in above commit.
I think we should stop the search at a commit boundary, e.g. the `commit:` line. I guess this is not a blocker.

Can we also make it work for the log view (e.g. `tig log -p`)?

One more thing: could you put the PR description in the commit message instead? The commit message will make it into the changelog (via `git shortlog`)